doc/IntlAPIs.md

```diff
@@ -49,27 +48,30 @@ One popular implementation strategy followed by other engines, is to bundle an i
   - `toLocaleDateString`
   - `toLocaleTimeString`
 
-## Not yet supported
-
-- [`Intl.PluralRules`](https://tc39.es/ecma402/#pluralrules-objects)
+## Supported Android only
+- `Intl.NumberFormat`
+  - `Intl.NumberFormat.prototype.formatToParts`
 
-- [`Intl.RelativeTimeFormat`](https://tc39.es/ecma402/#relativetimeformat-objects)
+## * Property support
 
-- [`Intl.DisplayNames`](https://tc39.es/proposal-intl-displaynames/#sec-intl-displaynames-constructor)
+### Limited iOS property support
+- `Intl.NumberFormat` implementation does not supporting the following properties,
+  - `notation`: 'compact'
+  - `notation`: 'engineering'
+  - `roundingType`
+  - `compactDisplay`
+  - `signDisplay`
 
-- [`Intl.ListFormat`](https://tc39.es/proposal-intl-list-format/#sec-intl-listformat-constructor)
+- `Intl.DateTimeFormat` implementation does not supporting the following properties,
+  - `numberingSystem`
 
-- [`Intl.Locale`](https://tc39.es/ecma402/#sec-intl-locale-constructor)
+### Limited Android property support
 
 - `Intl.DateTimeFormat` properties
    - [`dateStyle/timeStyle`](https://tc39.es/proposal-intl-datetime-style/)
```

Comment:
This is now supported as of #927

Reply:
Will move this to the `Android 7.0 - 7.1 and older (SDK < 24)` section just like #927 mentions.